packages/jest-cli/src/watch.js

```diff
@@ -138,12 +149,19 @@ export default function watch(
     }).catch(error => console.error(chalk.red(error.stack)));
   };
 
+  let activePlugin: ?WatchPlugin;
   const onKeypress = (key: string) => {
     if (key === KEYS.CONTROL_C || key === KEYS.CONTROL_D) {
       process.exit(0);
       return;
     }
 
+    if (activePlugin != null) {
```

Comment:
Not sure that @cpojer and @aaronabramov think but I feel that it would be interesting to internally implement `p` and `t`  as plugins so that they are the codebase just treat them as yet another plugin... It might also help to both use them as plugin references and also to always have something to validate and test that api